### PR TITLE
Reving up the Assembly version and package version of System.Console

### PR DESCRIPTION
**Pull request by joperezr**
cc: @weshaggard @ericstj @wtgodbe 

Revving up the Assembly and Package version of the System.Console package.

### CODE REVIEW

File: src/System.Console/dir.props [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.1.0</AssemblyVersion>
+    <AssemblyVersion>4.0.1.1</AssemblyVersion>
```
Comment (weshaggard) [inline src/System.Console/dir.props]: Similar question here whether or not we should bump the ref version as well.

File: src/System.Console/pkg/System.Console.pkgproj [hunk 2]
```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
```
Comment (weshaggard) [inline src/System.Console/pkg/System.Console.pkgproj]: You can put this in the dir.props.

Comment (joperezr) [reply src/System.Console/pkg/System.Console.pkgproj]: Good point, I'll move that over.

File: src/System.Console/dir.props (hunk 1 above)
Comment (joperezr) [reply src/System.Console/dir.props]: cc: @ericstj thoughts?

File: src/System.Console/pkg/System.Console.pkgproj [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
```
Comment (weshaggard) [inline src/System.Console/pkg/System.Console.pkgproj]: Not a big deal but you could also fix this by moving the dir.props file one level up.

**Pull request merged**